.github/workflows/ci.yml

```diff
@@ -250,20 +303,23 @@ jobs:
           ini-values: ${{ env.PHP_INI_VALUES }}
           tools: none
 
-      - name: Install java
-        uses: actions/setup-java@v1
+      - name: Download unscoped PHAR
+        uses: actions/download-artifact@v2
         with:
-          java-version: 1.8
+          name: phpunit-snapshot-unscoped-phar
+          path: ./build/artifacts/phpunit-snapshot.phar
 
       - name: Run regular test suite with unscoped PHAR
         run: ant run-regular-tests-with-unscoped-phar
 
-      - name: Run PHAR-specific end-to-end tests with scoped PHAR
-        run: ant run-phar-specific-tests-with-scoped-phar
+      - name: Remove unscoped PHAR
+        run: rm ./build/artifacts/phpunit-snapshot.phar
```

Comment:
Instead of using the same names for scoped and unscoped PHARs (and consequently, having to remove the unscoped PHAR here before downloading the scoped PHAR in the next step), we could use different names for the PHARs, but that would require updating `build.xml`.

What do you think?

Reply:
We do not ship unscoped PHARs (anymore), so there is not really a point in building and testing them. And as of 2f9ab3821055299950fc0d7853d5f5b82dd25791 we no longer do.

Reply:
👍